e2e/tests/api-driven/src/jwt.ts

```diff
@@ -0,0 +1,51 @@
+import { sign } from "jsonwebtoken";
+import { User, Role } from "@opensystemslab/planx-core/types";
+import { $admin } from "./client";
+
+// This code is copied from api.planx.uk/modules/auth/service.ts
```

Comment:
Not an ideal solution! Another candidate for functionality that could move to `planx-core` possibly?